roles/openshift_health_checker/openshift_checks/memory_availability.py

```diff
@@ -0,0 +1,31 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class MemoryAvailability(OpenShiftCheck):
+    """Check that recommended memory is available."""
+
+    name = "memory_availability"
+    tags = ["preflight"]
+
+    recommended_memory_mb = {
+        "nodes": 8000.0,
```

Comment:
I scanned over the output from `docker info` and saw that it reports different measurements in different prefix unit systems.

```
...
Storage Driver: devicemapper
 Pool Blocksize: 524.3 kB
 Base Device Size: 10.74 GB
 Data Space Used: 1.42 GB
 Data Space Total: 44.43 GB
 Data Space Available: 43.01 GB
 Metadata Space Used: 1.507 MB
 Metadata Space Total: 511.7 MB
 Metadata Space Available: 510.2 MB
 Thin Pool Minimum Free Space: 4.443 GB
...
Total Memory: 15.11 GiB
...
```

Sizes are reported using the SI base-10 system (`kB`, `MB`, `GB`) and memory is reported using the NIST base-2 system (`KiB`, `GiB`, etc).

As it was pointed out before, I do [maintain a library, 'bitmath'](http://bitmath.readthedocs.io/en/latest/index.html) (200 unit tests, in RHEL/Fedora, etc etc, etc) that handles all of that automatically (and more).

Here's a quick gist I made that takes code from this PR and shows the bitmath equivalent.
* [parse_docker_info.py](https://gist.github.com/tbielawa/a24316f6aa83a1c841220e4cb71f94ac)

I think that trying to get bitmath available as an RPM in the RHSM channels would be a big effort, and getting it into Atomic is nigh impossible. I'm OK w/ vendoring/slipping the bitmath library into this project.

But that's a bigger conversation we should probably have as a team.